code/modules/vore/appearance/sprite_accessories_taur_vr.dm

```diff
@@ -157,15 +157,27 @@
 	clip_mask_icon = 'icons/mob/vore/taurs_vr.dmi'
 	clip_mask_state = "taur_clip_mask_def" //Used to clip off the lower part of suits & uniforms.
 
-/datum/sprite_accessory/tail/taur/roiz_long_lizard // Not ACTUALLY a taur, but it uses 32x64 so it wouldn't fit in tails.dmi, and having it as a tail bugs up the sprite.
-	name = "Long Lizard Tail (Roiz Lizden)"
-	icon_state = "roiz_tail_s"
-	do_colouration = 0
-	ckeys_allowed = list("spoopylizz")
+// Species-unique long tails/taurhalves
+
+/datum/sprite_accessory/tail/taur/shadekin_tail
+	name = "Shadekin Tail (Shadekin)"
+	icon_state = "shadekin_s"
+	can_ride = 0
+	//ckeys_allowed = list("spoopylizz")
 	hide_body_parts = null
 	clip_mask_icon = null
```

Comment:
Sweet, glad to see these were nulled out. I saw this under the taur type and got a bit nervous but this was done properly.